BackofficeBundle/Resources/public/less/tokenInputPatches/token.less

```diff
@@ -0,0 +1,39 @@
+.tag.token-input-token-facebook {
```

Comment:
yes, but should duplicate entire theme code from tokenInput. Why not use the themeembedded with tokenInput? @itkg-alavieille